src/components/NodeId.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import { formatNodeId } from "@app/lib/utils";
+
+  import Avatar from "./Avatar.svelte";
+  import Hovercard from "./Hovercard.svelte";
+  import ModalStyling from "./ModalStyling.svelte";
+  import Clipboard from "./Clipboard.svelte";
+
+  export let nodeId: string;
+  export let alias: string | undefined = undefined;
+  export let disableTooltip: boolean = false;
+
+  export let styleColor: string | undefined = undefined;
+</script>
+
+<style>
+  .id {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 0.25rem;
+    height: 1rem;
+    font-family: var(--font-family-monospace);
+    font-weight: var(--font-weight-bold);
+    font-size: var(--font-size-small);
+    color: var(--color-fill-secondary);
+  }
+
+  .card {
+    height: 2.5rem;
+    white-space: nowrap;
+    font-family: var(--font-family-monospace);
+    font-weight: var(--font-weight-bold);
+    font-size: var(--font-size-small);
+    color: var(--color-fill-secondary);
+  }
+  .content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 0.25rem;
+  }
+</style>
+
+<Hovercard disabled={disableTooltip}>
+  <div slot="trigger" class="id" style:color={styleColor}>
+    <Avatar {nodeId} />
+    {#if alias}
+      {alias}
+    {:else}
+      {formatNodeId(nodeId)}
+    {/if}
+  </div>
+
+  <div slot="card" class="card">
+    <ModalStyling style="padding: 7px 1rem;">
```

Comment:
Hm, usually we try to avoid global classes, because it's sometimes hard to know where they come from, but maybe if we prefix it with `global-` it could be more obvious?

> In any case I would suggest to use the name „popover“ instead of „modal“. This is more in line with the usage of these terms in interface design. (A modal disables interaction with the rest of the app and forces the user to interact with the modal.)

bf9ca32d17ff6d78c80967f4a38843d0e380bebb

Reply:
You’re right, global classes should be prefixed to be able to distinguish them from classes in components.

I think we can leave it for now, but I would take another look at `Popover`, `HoverPopover` and `ModalStyling` because I noticed a couple of things:
1. For `NodeId` there are five nested containers before you get to the popover and they override various stuff
2. The names are inconsistent and confusing (“modal”, “toggle“, “trigger“, “card“)
3. `Popover` is always used in conjunction with `ModalStyling`
4. You can often style the element contained in `ModalStyling` instead of `ModalStyling` (`padding` and `width`)
5. You should probably use a library for positioning the popovers like https://www.npmjs.com/package/svelte-popperjs

Reply:
> 2. The names are inconsistent and confusing (“modal”, “toggle“, “trigger“, “card“)

0cbc1ac9468d8c338c9afd23117655877f8aa99a

Reply:
> 1. For NodeId there are five nested containers before you get to the popover and they override various stuff

393f6b83a90d68544222ea0ab5c63d0b9b53b342

Reply:
> 3. Popover is always used in conjunction with ModalStyling
> 4. You can often style the element contained in ModalStyling instead of ModalStyling (padding and width)

Got rid of the `ModalStyling` component by moving the styling into `Popover` and `HoverPopover` 0878e593e947c9c6e7d28e0f34b4170b4ae71634.

Reply:
> 5. You should probably use a library for positioning the popovers like https://www.npmjs.com/package/svelte-popperjs

Yeah I agree. Or we write a component that does the positioning for us.